google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -176,6 +202,8 @@ public Long getUniformBucketLevelAccessLockedTime() {
               : new DateTime(uniformBucketLevelAccessLockedTime));
 
       iamConfiguration.setUniformBucketLevelAccess(uniformBucketLevelAccess);
+      iamConfiguration.setPublicAccessPrevention(
+          publicAccessPrevention == null ? null : publicAccessPrevention.getValue());
```

Comment:
I'm not sure this case will ever be relevant, as the default value for `publicAccessPrevention` is "unspecified". This field should never be null.

Reply:
If a request uses a field selector which does not include iamConfiguration, then this value will be null because it's not returned by the service.